src/map/status.c

```diff
@@ -9911,145 +9925,153 @@ int status_change_start(struct block_list *src, struct block_list *bl, enum sc_t
 
 	/* values that must be set regardless of SCFLAG_LOADED e.g. val_flag */
 	switch(type) {
-		case SC_FIGHTINGSPIRIT:
```

Comment:
please and this formatting changes not related to clans. Can you move it into separate commit?